scripts/load_csv_psql.py

```diff
@@ -41,28 +42,35 @@
         default=';')
     parser.add_argument(
         '--host',
-        help='Database host',
-        default=environ['DB_HOST'])
+        help='Database host')
     parser.add_argument(
         '--user',
-        help='Database username',
-        default=environ['DB_USER'])
+        help='Database username')
     parser.add_argument(
         '--password',
-        help='Database user password',
-        default=environ['DB_PASSWORD'])
+        help='Database user password')
     parser.add_argument(
         '--port',
         help='Database connection port',
         default=5432)
-
     args = parser.parse_args()
+
+    if not args.host:
+        args.host = environ['DB_HOST']
+    if not args.user:
+        args.user = environ['DB_USER']
+    if not args.password:
+        args.password = environ['DB_PASSWORD']
```

Comment:
@RuggeroFabbiano this conversation doesn't show where those arguments were before my change, so I can't respond fully. I remember I had initially set them as `default` in the `ArgumentParser` and then I got error because it tried to evaluate `environ[...]` before checking if the default was needed or not. A bit like the other comment of yours above.